server/springboot/build.gradle

```
@@ -0,0 +1,51 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '3.3.0'
+	id 'io.spring.dependency-management' version '1.1.4'
+}
+
+group = 'com.styra'
+version = '0.0.1-SNAPSHOT'
+
+java {
+	sourceCompatibility = '17'
+    targetCompatibility = '17'
```

```suggestion
	sourceCompatibility = '17'
	targetCompatibility = '17'
```
[nit]: Possible typo?

---

I assume you were just pointing out the messed up indents? I have fixed those.